netsim/devices/dellos10.yml

```diff
@@ -23,6 +23,7 @@ features:
   gateway:
     protocol: [ anycast, vrrp ]
   lag:
+    requires: [ vlan ]  # Need to configure 'switchport' before certain (m)lag settings
```

Comment:
This is wrong. If LAG needs extra configuration on Dell OS10, then the LAG template should provide that configuration, unless you're telling me that you cannot have pure routed LAG on OS10.

Reply:
Removed